web-console/src/lib/services/manager/schemas.gen.ts

```diff
@@ -1945,6 +1945,15 @@ as long as different pipelines running on the same machine are pinned to
 different CPUs.`,
           default: []
         },
+        provisioning_timeout_s: {
```

Comment:
_seconds

Reply:
Changed to `_secs` to keep consistent with other existing runtime configuration parameters.